pkg/testutils/kyma.go

```diff
@@ -50,7 +49,7 @@ func NewKymaWithNamespaceName(name, namespace, channel string) *v1beta2.Kyma {
 	return builder.NewKymaBuilder().
 		WithNamePrefix(name).
 		WithNamespace(namespace).
-		WithAnnotation(watcher.DomainAnnotation, "example.domain.com").
+		WithAnnotation(shared.SKRDomainAnnotation, "example.domain.com").
```

Comment:
maybe also rename to `SkrDomainAnnotation` to have consistency.

Reply:
https://github.com/kyma-project/lifecycle-manager/pull/2382/commits/541044693eff4e4e48d95750d9eff72b8bf008e4